lib/pf/fingerbank.pm

```diff
@@ -100,14 +99,9 @@ sub _query {
     $mac = substr($mac, 0, 6);  # Only keep first 6 characters (OUI)
     $cached_args->{'mac'} = $mac;
 
-    return $cache->compute_with_undef($cached_args, 
-        sub {
-            $logger->debug("Fingerbank result not in cache (either not present or expired). Querying Fingerbank for result");
-            my $fingerbank = fingerbank::Query->new;
-            return $fingerbank->match($args);
-        },
-        {expires_in => FINGERBANK_CACHE_EXPIRE}
-    );
+    my $cache = pf::CHI->new( namespace => 'fingerbank', expires_in => FINGERBANK_CACHE_EXPIRE );
```

Comment:
Perfect.

I added a note to #1041 to remove this once its merged (as I doubt it will be in maintenance)